lib/SPIRV/libSPIRV/SPIRVValue.cpp

```diff
@@ -51,6 +51,14 @@ void SPIRVValue::setAlignment(SPIRVWord A) {
     eraseDecorate(DecorationAlignment);
     return;
   }
+  SPIRVWord PrevAlignment;
```

Comment:
This might not be a relevant issue. But I will go ahead and ask. If we had A==0 and the prevAlignment != 0, do we want to error out or just delete the decoration?

Thanks

Reply:
The current behaviour with `A == 0` is to remove all existing Alignment decorations and my patch doesn't change this.  I think that behaviour is fine for now.